modules/microshift-greenboot-dir-structure.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: CONCEPT
+[id="microshift-greenboot-dir-structure_{context}"]
+= How greenboot uses directories to run scripts
```

Comment:
Isn't this documented in the RHEL docs? Maybe we can just focus on how MicroShift integrates with Greenboot here in the MicroShift docs?

Reply:
I am not familiar with the RHEL docs. Whatever is specific to MicroShift is fine with me, as long as there is enough context without a customer having to leap to another docset.